docs/serviceability.md

```diff
@@ -0,0 +1,58 @@
+# Serviceability
```

Comment:
What kind of info are we looking for on 1 and 2 in your suggestion..

Reply:
Just an overview blurb describing the purpose and intent of this doc.  When I read this doc without any context,  I see it can serve two purposes 1.  From a developer perspective for someone who is unfamiliar with instrumenting logs and what is supposed to go into them, the `Understanding the Logs` topic can be useful.  2) from the troubleshooting perspective for a developer/SRE.   Understanding where to look for the logs and what to do with them is also useful.

Reply:
Updated!